CONTRIBUTING.md: Fix miscellaneous typos (#245)
golioth/golioth-zephyr-sdk: CONTRIBUTING.md

<!-- omit in toc -->
### How Do I Submit a Good Bug Report?

> > It is *vitally* important to NOT publicly report security related issues, vulnerabilities or bugs. Instead sensitive bugs must be sent by email to <security@golioth.io>. We take security very seriously,appreciate your research and will work with you to resolve the situation ASAP.

Reviewer: ```suggestion
> We take security very seriously. It is *vitally* important to NOT publicly report security related issues, vulnerabilities or bugs. Instead, please send your findings by email to <security@golioth.io>. We appreciate your research and will work with you to resolve the situation ASAP.
```
